docs/ad/api.md

```diff
@@ -151,6 +138,133 @@ Options | Description | Type | Required
 }
 ```
 
+To set a category field for high cardinality:
+
+#### Request
+
+```json
+POST _opendistro/_anomaly_detection/detectors
+{
+  "name": "Second Test Detector 2",
+  "description": "ok rate",
+  "time_field": "@timestamp",
+  "indices": [
+    "host-cloudwatch"
+  ],
+  "category_field": [
+    "host"
+  ],
+  "feature_attributes": [
+    {
+      "feature_id": "latency_max",
+      "feature_name": "latency_max",
+      "feature_enabled": true,
+      "importance": 1,
+      "aggregation_query": {
+        "latency_max": {
+          "max": {
+            "field": "latency"
+          }
+        }
+      }
+    }
+  ],
+  "window_delay": {
+    "period": {
+      "interval": 10,
+      "unit": "SECONDS"
+    }
+  },
+  "detection_interval": {
+    "period": {
+      "interval": 1,
+      "unit": "MINUTES"
+    }
+  },
+  "ui_metadata": {
+    "aabb": {
+      "ab": "bb"
+    }
+  },
+  "schema_version": 2
```

Comment:
User shouldn't input `schema_version`. AD backend should fill this field with current latest schema version automatically in case user input some wrong value which may cause migration failure in future.